frontend/plugin-manager/OBSPluginManager.cpp

```diff
@@ -0,0 +1,41 @@
+#include "ObsPluginManager.hpp"
+
+#include <OBSApp.hpp>
+#include <QVBoxLayout>
+#include <QHBoxLayout>
+#include <QCheckBox>
+#include <QScrollArea>
+#include <QDialogButtonBox>
+#include <QLabel>
+
+#include "moc_OBSPluginManager.cpp"
+OBSPluginManager::OBSPluginManager(std::vector<OBSModuleInfo> const& modules, QWidget* parent)
+	: QDialog(parent), _modules(modules), ui(new Ui::OBSPluginManager)
+{
+	setWindowFlags(windowFlags() & ~Qt::WindowContextHelpButtonHint);
+
+	ui->setupUi(this);
+
+	std::sort(_modules.begin(), _modules.end(), [](const OBSModuleInfo& a, const OBSModuleInfo& b) {
+		auto aName = a.display_name != "" ? a.display_name : a.module_name;
+		auto bName = b.display_name != "" ? b.display_name : b.module_name;
+		return aName < bName;
+	});
+
+	for (auto& metadata : _modules) {
+		std::string name = metadata.display_name != "" ? metadata.display_name : metadata.module_name;
+		auto item = new QListWidgetItem(name.c_str());
```

Comment:
Does `QListWidgetItem` have a constructor that takes a `const char *` or does this rely on automatic conversion into `QString`?